doc/MANUAL.adoc

```diff
@@ -314,10 +314,47 @@ this:
    `$HOME/.ccache/ccache.conf`.
 5. Otherwise, if `XDG_CONFIG_HOME` is set then use
    `$XDG_CONFIG_HOME/ccache/ccache.conf`.
-6. Otherwise, use `%APPDATA%/ccache/ccache.conf` (Windows),
+6. Otherwise, use
    `$HOME/Library/Preferences/ccache/ccache.conf` (macOS) or
    `$HOME/.config/ccache/ccache.conf` (other systems).
 
+The location of the cache directory is determined as follows:
+
+1. `cache_dir` in the above found configuration if specified.
+2. `$CCACHE_DIR` if set.
+3. `$HOME/.ccache` if it exists.
+4. `$XDG_CACHE_HOME/ccache` if `$XDG_CACHE_HOME` is set.
+5. Otherwise `~/.cache/ccache` or `~/Library/Caches/ccache` on Apple systems.
```

Comment:
This is already described for _cache_dir (CCACHE_DIR)_ in the _Configuration options_ section. Would it be enough to refer to that description to avoid having two descriptions?

Reply:
Since I made the section "config and cache locations", is it ok if I do the opposite and link the `ccache_dir` option to this section.

Reply:
I see `cache_dir`/`CCACHE_DIR` as a configuration option like any other configuration option, so unless you think that it would be significantly better to describe it together with the configuration file location logic I would prefer keeping the description in _Configuration options_.

(In contrast, the configuration file location needs to be described separately since it's not a configuration option.)

Reply:
Ok, I will expand the `cache_dir` description and make it more accurate and link to it from the configuration file section.

Reply:
How's this?

Reply:
:+1: